profiler/README.MD

```diff
@@ -50,21 +46,7 @@ To build C++ projects
 - git
 
 #### Building from the command line
-1. Build C# projects
```

Comment:
Just my 2 cents, but I think we should be encouraging the use of Nuke, as that's the only thing that's really tested.

Reply:
This is the VS driven way to compile and debug.
Feel free to add the corresponding NUKE way to build!